libraries/fbx/src/GLTFSerializer.cpp

```diff
@@ -1621,12 +1624,14 @@ bool GLTFSerializer::buildGeometry(HFMModel& hfmModel, const hifi::VariantHash&
                 }               
             }
 
-            // Add epsilon to mesh extents to compensate for planar meshes
-            mesh.meshExtents.minimum -= glm::vec3(EPSILON, EPSILON, EPSILON);
-            mesh.meshExtents.maximum += glm::vec3(EPSILON, EPSILON, EPSILON);
-            hfmModel.meshExtents.minimum -= glm::vec3(EPSILON, EPSILON, EPSILON);
-            hfmModel.meshExtents.maximum += glm::vec3(EPSILON, EPSILON, EPSILON);
-           
+            // Mesh extents must be at least minimum entity size, in particular for blendshapes to work on planar meshes.
+            const float ENTITY_ITEM_MIN_DIMENSION = 0.001f;
```

Comment:
"Entity" isn't really appropriate here since this code is agnostic of how the model is eventually used.

Reply:
Setting the minim dimension to the minimum entity dimension solves the problem of applying blendshapes to planar meshes. Using the original `EPSILON` value results in external code rescaling the mesh to the minimum entity dimension, i.e., multiplying the plane dimension by `ENTITY_ITEM_MIN_DIMENSION / EPSILON = 0.001 / 0.000001f = 1000.0`, wiht the result that the blendshape  distortions are also multiplied by `1000.0` in this direction.
This is the "morph plane" test case.

Reply:
I see.  Can we just rename ENTITY_ITEM_MIN_DIMENSION to something like MODEL_MIN_DIMENSION?

I also wonder how FBX handles this case.  are planar blendshapes similarly broken there?

Reply:
I will rename as suggested.
A planar FBX model may very well suffer the same problem. I can't see in the FBX code where planar mesh is handled. I've added an issue to test/fix the FBX case: https://github.com/kasenvr/project-athena/issues/675